build/scripts/ImportTestDataInBcContainer.ps1

```diff
@@ -0,0 +1,36 @@
+Param(
+    [Hashtable]$parameters
+)
+
+Import-Module $PSScriptRoot\AppExtensionsHelper.psm1
+
+function Invoke-ContosoDemoTool() {
+    param(
+        [string]$ContainerName,
+        [string]$CompanyName = (Get-NavDefaultCompanyName),
+        [switch]$SetupData = $false,
+        [string]$DemodataApp = "Contoso Coffee Demo Dataset"
```

Comment:
You will probably need a few downstream apps as well (search for `ContosoDemoData` in `groups.json`).

They are not explicitly called but the code is run by subscribing some events.

---

We have an upcoming discussion about the implicit dependency between tests and demo data.

Reply:
Moved some logic around now such that all the demo data apps are installed when we generate demo data